sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3267,6 +3267,11 @@ object functions {
   /**
    * Returns an array containing all the elements in `x` from index `start` (or starting from the
    * end if `start` is negative) with the specified `length`.
+   *
+   * @param x the column which is to be sliced
+   * @param start the starting index of the slice
+   * @param length the desired length of the slice
```

Comment:
Why does it has the different statement from the one above? Can you make it consistent?